docs/rst/installation/linux_installation.rst

```diff
@@ -2,3 +2,337 @@
+
+    .. code-block:: bash
+
+        # Python3 installation
+        sudo apt install python3.10-venv
+
+        # Virtual environment deployment
+        cd <path_to_virtual_environment_directory>
+        python3 -m venv fastdds_qosprofman_venv
+        source fastdds_qosprofman_venv/bin/activate
+
+        # Required dependencies installation
+        pip3 install -r ~/fastdds_qosprofman_ws/src/Fast-DDS-QoS-Profiles-Manager/docs/requirements.txt
+
+    .. note::
+
+        The ``<path_to_virtual_environment_directory>`` could be inside the workspace directory if project is built using CMake.
+        As long as colcon performs a recursive search to build packages inside the workspace directory, it is recommended to deploy the virtual environment outside of the workspace directory in order to avoid conflicts.
+
+            .. tabs::
+
+                .. tab:: Building with colcon
+
+                        .. code-block:: bash
+
+                            # Virtual environment deployment
+                            cd ~
+
+                .. tab:: Building with CMake
+
+                        .. code-block:: bash
+
+                            # Virtual environment deployment
+                            cd ~/fastdds_qosprofman_ws
+
+.. _installation_download_project_source_linux:
+
+Tool suite build process
+------------------------
+
+Create a workspace directory and download the project:
+
+.. code-block:: bash
+
+    mkdir -p ~/fastdds_qosprofman_ws/src
+    cd ~/fastdds_qosprofman_ws/src
+    git clone https://github.com/eProsima/Fast-DDS-QoS-Profiles-Manager.git
+
+.. _installation_build_source_linux:
+
+.. tabs::
+
+    .. tab:: Building with colcon
+
+        *eProsima Fast DDS QoS Profiles Manager* complete tool suite is build using `colcon <https://colcon.readthedocs.io/en/released/>`_  by running:
+
+        .. code-block:: bash
+
+            cd ~/fastdds_qosprofman_ws
+            colcon build
+
+        .. note::
+
+            Being based on `CMake <https://cmake.org/>`_ it is possible to pass `CMake configuration options <https://colcon.readthedocs.io/en/released/reference/verb/build.html#cmake-specific-arguments>`_ to the :code:`colcon build` command.
+            These options can be also passed through a `colcon.meta <https://colcon.readthedocs.io/en/released/user/configuration.html?highlight=colcon.meta#using-meta-files>`_ file.
+
+            For instance, the *Library* and *Documentation* test suites building is enabled by building with the CMake option `EPROSIMA_BUILD_TESTS`.
+            If GTest repository has been included in the workspace directory, add the GTest dependency (``"googletest-distribution"``) in both *Library* and *Documentation* `colcon.pkg <https://colcon.readthedocs.io/en/released/user/configuration.html?highlight=colcon.meta#colcon-pkg-files>`_ files.
```

Comment:
As the documentation depends on the library, it is enough adding it only to the library package. I think we should keep this as simple as possible.